### ISSUE TITLE
CVPR'22 Camera Ready Activities Tracking

### ISSUE THREAD

**Issue by taiya** [google-research/kubric#184]
**Camera ready deadline: March 28th**

Most important is to have the `gs://kubric-public/...` bucket ready, with a `kubric/challenges/name_your_challenge/README.md` describing its content. The worker script that generates is is expected, but not due on Monday.

# Code 

- [x] fix github tests @taiya
- [x] fix pylint @taiya 

# Challenges

- [x] Section 4.1 (MOVi-A..E) *[@Qwlouse]*
  - [x] worker script
  - [ ] ~~baseline/metric~~
  - [x] MOVi-A
  - [x] MOVi-B (ETA: Match 22nd)
  - [x] MOVi-C (ETA: Match 22nd)
  - [x] MOVi-D (ETA: Match 22nd)
  - [x] MOVi-E (ETA: Match 28th)

- [x] Section 4.2 (MOVi-F) *[@Qwlouse]*
  - [x] worker script (path?) 
  - [x] dataset to public gs:// (ETA: March 28th)

- [ ] Section 4.3 Texture Structure in NeRF *[@drebain,@taiya]* 
  - [ ] `challenges/texture_structure_nerf/README.md`
  - [x] `challenges/texture_structure_nerf/worker.py`
  - [x] `gs://kubric-public/data/texture_structure_nerf`
 
- [x] Section 4.5 Pre-training visual representations *[@Qwlouse]*
  - [x] `challenges/pretraining_visual/README.md`
  - [x] `challenges/pretraining_visual/worker.py`
  - [x] `gs://kubric-public/data/pretraining_visual`

- [ ] Appendix B.1 Robust NeRF *[@sarasra,@taiya]*
  - [ ] `challenges/robust_nerf/README.md`
  - [x] `challenges/robust_nerf/worker.py`
  - [x] `gs://kubric-public/data/robust_nerf`

- [x] Appendix B.2 Multi-view object matting *[@ziyuw,@taiya]*
  - [x] `challenges/multiview_matting/README.md`
  - [x] `challenges/multiview_matting/worker.py`
  - [x] `gs://kubric-public/data/multiview_matting`

- [ ] Appendix B.3 Complex BRDFs *[@yilundu,@taiya]*
  - [ ] `challenges/complex_brdf/README.md`
  - [x] `challenges/complex_brdf/worker.py`
  - [x] `gs://kubric-public/data/complex_brdf`

- [ ] Appendix B.4 Single View Reconstruction  *[@IssamLaradji]*
  - [ ] `challenges/single_view_reconstruction/README.md`
  - [x] `challenges/single_view_reconstruction/worker.py`
  - [x] `gs://kubric-public/data/single_view_reconstruction`
 
- [x] Appendix B.5 Video Based Reconstruction *[@ChikaYan,@taiya]*
  - [x] `challenges/video_based_reconstruction/README.md`)
  - [x] `challenges/video_based_reconstruction/worker.py`)
  - [x] `gs://kubric-public/data/video_based_reconstruction`
 
- [x] Appendix B.6 Point Tracking *[@cdoersch, @Qwlouse]*
  - [x] `challenges/point_tracking/README.md`
  - [x] ~~`challenges/point_tracking/worker.py`~~  (uses MOVi-E data)
  - [x] ~~`gs://kubric-public/data/point_tracking`~~ (uses MOVi-E data)

**Comment by drebain** [comment]
PR #182 is the worker for the NeRF texture data. I also have the data, but I'm not sure how to upload it from a university machine.

**Comment by taiya** [comment]
Please merge #182 (+test), sync with @Qwlouse for access to the "dropbox" bucket.
